### ISSUE TITLE
On failure, error description is missing.

### ISSUE THREAD

**Issue by joseinweb** [stagingrdkm/LISA#45]
Pre Requisite : Install_Test_Application
Pre Requisite No : 1
TEST STEP NAME   :  Check_Application_List
TEST STEP ID     :  1
PLUGIN API NAME  :  LISA.1.getList
EXPECTED VALUES  :  sleepy_bundle
TEST STEP STATUS :  SUCCESS
----------------- Result -----------------
APPSTATUS        :  TRUE
#--------- [Pre-requisite Status] : SUCCESS ----------#
Test Case Pre-requisite Status: SUCCESS

JSON Cmd :  {"params": {"category": "category", "version": "1.0", "appName": "sleepy_bundle", "url": "https://testtools.rdkcentral.com:8443/rdk-test-tool/images/sleepy_bundle.tar.gz", "type": "dac", "id": "sleepy_bundle"}, "jsonrpc": "2.0", "id": 2, "method": "LISA.1.install"}
Response :  {u'jsonrpc': u'2.0', u'id': 2, u'error': {u'message': u'', u'code': 1003}}
TEST STEP NAME   :  Install_Application
TEST STEP ID     :  1
PLUGIN API NAME  :  LISA.1.install
INPUT PARAMETER  :  category - category, version - 1.0, appName - sleepy_bundle, url - https://testtools.rdkcentral.com:8443/rdk-test-tool/images/sleepy_bundle.tar.gz, type - dac, id - sleepy_bundle
EXPECTED VALUES  :  already installed,1003
TEST STEP STATUS :  FAILURE
----------------- Result -----------------
ERROR            :  {u'message': u'', u'code': 1003}
##--------- [TEST EXECUTION STATUS] : FAILURE

TEST CASE ID     :  LISA_09
DESCRIPTION      :  Checks the error message on uninstalling the non existing application
#==============================================================================#
#-------------- Test Case Pre-Requisite ---------------#
Pre Requisite : UnInstall_Test_Application
Pre Requisite No : 1
[INFO]: Received empty JSON response result
TEST STEP NAME   :  Check_Application_List
TEST STEP ID     :  1
PLUGIN API NAME  :  LISA.1.getList
EXPECTED VALUES  :  sleepy_bundle
TEST STEP STATUS :  SUCCESS
----------------- Result -----------------
APPSTATUS        :  FALSE
#--------- [Pre-requisite Status] : SUCCESS ----------#
Test Case Pre-requisite Status: SUCCESS

JSON Cmd :  {"params": {"uninstallType": "full", "type": "dac", "version": "1.0", "id": "sleepy_bundle"}, "jsonrpc": "2.0", "id": 2, "method": "LISA.1.uninstall"}
Response :  {u'jsonrpc': u'2.0', u'id': 2, u'error': {u'message': u'', u'code': 1001}}
TEST STEP NAME   :  UnInstall_Application
TEST STEP ID     :  1
PLUGIN API NAME  :  LISA.1.uninstall
INPUT PARAMETER  :  uninstallType - full, type - dac, version - 1.0, id - sleepy_bundle
EXPECTED VALUES  :  wrong parameters,1001
TEST STEP STATUS :  FAILURE
----------------- Result -----------------
ERROR            :  {u'message': u'', u'code': 1001}
##--------- [TEST EXECUTION STATUS] : FAILURE

TEST CASE ID     :  LISA_10
DESCRIPTION      :  Checks the error message on installing invalid app
#==============================================================================#
JSON Cmd :  {"params": {"category": "category", "version": "1.0", "appName": "sleepy_bundle", "url": "https://testtools.rdkcentral.com:8443/rdk-test-tool/images/sleepy_bundle.tar.gz", "type": "dac", "id": ""}, "jsonrpc": "2.0", "id": 2, "method": "LISA.1.install"}
Response :  {u'jsonrpc': u'2.0', u'id': 2, u'error': {u'message': u'', u'code': 1001}}
TEST STEP NAME   :  Install_Application
TEST STEP ID     :  1
PLUGIN API NAME  :  LISA.1.install
INPUT PARAMETER  :  category - category, version - 1.0, appName - sleepy_bundle, url - https://testtools.rdkcentral.com:8443/rdk-test-tool/images/sleepy_bundle.tar.gz, type - dac, id - 
EXPECTED VALUES  :  request not accepted,1001
TEST STEP STATUS :  FAILURE
----------------- Result -----------------
ERROR            :  {u'message': u'', u'code': 1001}
##--------- [TEST EXECUTION STATUS] : FAILURE

**Comment by sverkoye** [comment]
Yes, indeed the error message is empty. After some investigation IMHO it looks like there is no easy solution. LISA functions return an error code but I don’t see a way to return the related message. wpeframework does fill in a message for error codes that it knows itself (the common error codes): https://github.com/rdkcentral/Thunder/blob/89968f149299ff5647cc4a4ab9d4f948c858ebff/Source/WPEFramework/PluginServer.h#L1559
https://github.com/rdkcentral/Thunder/blob/89968f149299ff5647cc4a4ab9d4f948c858ebff/Source/core/Portability.h#L835
